### PR TITLE
chore: add Jenkins config and maintainer

### LINKED ISSUE #17: Should be part of the IPFS org + needs a Lead Maintainer + needs to have Jenkins run.

### PR DESCRIPTION
**Pull request by lidel**
Closes #17 

@diasdavid  I don't have admin on this repo, can't move it to /ipfs/ org, pls help
@VictorBjelkholm this PR adds Jenkinsfile, but I assume someone needs to whitelist this repo at Jenkins manually?

### CODE REVIEW

**Comment by diasdavid** [comment]
@lidel moved :)

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 is-ipfs
 ====
 
-[![build status](https://secure.travis-ci.org/ipfs/is-ipfs.svg)](http://travis-ci.org/ipfs/is-ipfs)
-[![dignified.js](https://img.shields.io/badge/follows-dignified.js-blue.svg?style=flat-square)](https://github.com/dignifiedquire/dignified.js)
+[![](https://img.shields.io/github/release/ipfs/ipfs-companion.svg)](https://github.com/ipfs/ipfs-companion/releases/latest)
```
Comment (diasdavid) [inline README.md]: IPFS companion?

Comment (lidel) [reply README.md]: Ooops, fixed in https://github.com/ipfs/is-ipfs/pull/18/commits/ae0f73809dc55989b5465626c30bdd9ad147e4ef

**Comment by lidel** [comment]
Superseded by #20

**Pull request closed without merging**